### DISCUSSION TITLE
The Chinese characters are garbled when deploying jfreechart on Kubernetes.

### DISCUSSION THREAD

**Discussion by Qinssai** [jfree/jfreechart#401] [Q&A] [answered]
When I am developing locally, the Chinese characters in the chart display correctly without any garbling. However, after deploying the application using Kubernetes (K8s), the Chinese characters in the chart become garbled.
![example](https://github.com/jfree/jfreechart/assets/41543199/1a78794f-ca66-414b-9271-f33cbb7691a1)

**Comment by trashgod** [comment]
Check the server's `Locale`, for [example](https://stackoverflow.com/q/8809098/230513); if it's a missing `Font`, this [Q&A](https://stackoverflow.com/q/12112353/230513) may provide some insight.

**Reply by Qinssai** [reply, marked as answer]
Thank you very much. I have found a solution. The reason for this problem is that the font I use is missing in the basic image. I remade the basic image, and the problem has been solved.
